force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -147,7 +170,8 @@ public with sharing class VariablesDataTypesOperators {
      * @return True if the date is today or in the future, False otherwise.
      */
     public static Boolean isDateTodayOrFuture(Date dt) {
-        return null; // Replace null with the variable you used to store the result
+        Date todaysDate = Date.today();
+        return dt.daysBetween(todaysDate) < 0; // Replace null with the variable you used to store the result
```

Comment:
This is one way to do it, but you can also do what you did in the method above and return a comparison to today's date instead of calculating the days between.